test/e2e/specs/realrp/vnetpeering.go

```diff
@@ -0,0 +1,129 @@
+package realrp
+
+import (
+	"context"
+	"errors"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-06-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp/client"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+)
+
+var _ = Describe("Peer Vnet tests [Vnet]", func() {
+	var (
+		ctx          = context.Background()
+		cli          *azure.Client
+		rg           = os.Getenv("RESOURCEGROUP")
+		vnetPeerName = "vnetPeer"
+		region       = os.Getenv("AZURE_REGION")
+		clientID     = os.Getenv("AZURE_CLIENT_ID")
+		clientSecret = os.Getenv("AZURE_CLIENT_SECRET")
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = azure.NewClientFromEnvironment(false)
+		Expect(err).ToNot(HaveOccurred())
+		if os.Getenv("AZURE_REGION") == "" {
+			Expect(errors.New("AZURE_REGION is not set")).ToNot(HaveOccurred())
+		}
+		if os.Getenv("RESOURCEGROUP") == "" {
```

Comment:
I guess we could simplify this to `Expect(rg).NotTo(BeEmpty())`

Reply:
The error message from this is bad:
```
  Expected
      <string>: 
  not to be empty
```
We should either create a wrapper for this like:
```go
func ensureEnvExists(env string) {
  if os.Getenv(env) == "" {
	Expect(fmt.Errorf("%s is not set", env)).ToNot(HaveOccurred())
  } 
}
```
or use `envconfig` but that would need as to centralize config into a single struct or set of structs.